### PR TITLE
Bha

### PR DESCRIPTION
**Pull request by BhagyashriSSavant**

### CODE REVIEW

File: Bhagyashri/Correction/css/style.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
+
+body{
+background-color:#006966;
+padding-left:100px;
+padding-right:100px;
+margin:0px;
```
Comment (sachinghuge24) [inline Bhagyashri/Correction/css/style.css]: add padding: 0 100px;
margin: 0;
whenever we are using 0 we don't need unit.

File: Bhagyashri/Correction/css/style.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	}
+nav{
+	color:#008080;
+	padding:10px;
+}	
+
+a{
+	text-decoration:none;
+	padding-left:10px;
+	padding-right:10px;
```
Comment (sachinghuge24) [inline Bhagyashri/Correction/css/style.css]: padding: 0 10px;

File: Bhagyashri/Correction/css/style.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+ }
+ .class-h3{
+	padding-top:30px;
+	padding-bottom:30px;
+ }
```
Comment (sachinghuge24) [inline Bhagyashri/Correction/css/style.css]: padding: 30px 0;

File: Bhagyashri/Correction/css/style.css [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+.class-h31{
+
+	margin-bottom:4px;
+	text-align:center;
+	font-size=20px;
```
Comment (sachinghuge24) [inline Bhagyashri/Correction/css/style.css]: font-size: 20px; 
you had use equal to operator here.

File: Bhagyashri/Correction/css/style.css [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+	color:#ffffff;
+	
+}
+.class-h32{
+
+	padding-top:0px;
+	border-top:0px;
+	margin-top:0px;
+    text-align:center;
+	font-size=20px;
```
Comment (sachinghuge24) [inline Bhagyashri/Correction/css/style.css]: same you had use the equal to operator here.
border-top:0px not needed here.

File: Bhagyashri/Correction/css/style.css [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+	border-style:normal;
+	border-color:none;
+	background-color:#007070;
+    background-color:rgba(00,70,70,0.2);
+    color:rgba(00,40,40);
+     }
+                
+.class-first{
+	 min-height:90px;
+	 Position:relative;
```
Comment (sachinghuge24) [inline Bhagyashri/Correction/css/style.css]: P is capital here.

File: Bhagyashri/Correction/css/style.css [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+}
+	 
+.div-right img{
+     height:300px;
+	 width:500px;
```
Comment (sachinghuge24) [inline Bhagyashri/Correction/css/style.css]: for image you can add width: 100%; and add min-height: 300px;
don't add fix height.

File: Bhagyashri/Correction/index.html [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+        <section>
+          <div class="class-left">
+	         <h1>Featured Project</h1>
+	         <h2>Project Name</h2>
+	      <div>
+	          <p>Lorem Ipsum is simply dummy text of the printing and<br/> 
+		         typesetting industry.Lorem Ipsum has been the industri's <br/>
+		         standard dummy text ever since.
+	          </p>
+		       <ul type="none">
```
Comment (sachinghuge24) [inline Bhagyashri/Correction/index.html]: remove type attribute here. and add css attribute to ul
list-type: none;

File: Bhagyashri/first_assignment/css/style.css [hunk 9]
```diff
@@ -0,0 +1,108 @@
+
+body{
+background-color:#006966;
+padding-left:100px;
+padding-right:100px;
```
Comment (sachinghuge24) [inline Bhagyashri/first_assignment/css/style.css]: merge those padding to single line.

File: Bhagyashri/first_assignment/css/style.css [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+	}
+nav{
+	color:#008080;
+	padding:10px;
+}	
+
+a{
+	text-decoration:none;
+	padding-left:10px;
+	padding-right:10px;
```
Comment (sachinghuge24) [inline Bhagyashri/first_assignment/css/style.css]: merge to single line.

File: Bhagyashri/first_assignment/css/style.css [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+ .class-h3{
+	padding-top:30px;
+	padding-bottom:30px;
+ }
+
+.class-h31{
+
+	margin-bottom:4px;
+	text-align:center;
+	font-size=20px;
```
Comment (sachinghuge24) [inline Bhagyashri/first_assignment/css/style.css]: equal to sign in css is not allowed.

File: Bhagyashri/first_assignment/home.html [hunk 12]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>home page</title>
+	<style>
```
Comment (sachinghuge24) [inline Bhagyashri/first_assignment/home.html]: why this inline css added here.

File: Bhagyashri/Correction/css/style.css (hunk 1 above)
Comment (BhagyashriSSavant) [reply Bhagyashri/Correction/css/style.css]: okay

File: Bhagyashri/Correction/css/style.css (hunk 3 above)
Comment (BhagyashriSSavant) [reply Bhagyashri/Correction/css/style.css]: done

File: Bhagyashri/Correction/css/style.css (hunk 4 above)
Comment (BhagyashriSSavant) [reply Bhagyashri/Correction/css/style.css]: changed

File: Bhagyashri/Correction/index.html (hunk 8 above)
Comment (BhagyashriSSavant) [reply Bhagyashri/Correction/index.html]: Have added list-style:none;

File: Bhagyashri/first_assignment/css/style.css (hunk 9 above)
Comment (BhagyashriSSavant) [reply Bhagyashri/first_assignment/css/style.css]: done

File: Bhagyashri/first_assignment/css/style.css (hunk 10 above)
Comment (BhagyashriSSavant) [reply Bhagyashri/first_assignment/css/style.css]: done;

File: Bhagyashri/first_assignment/css/style.css (hunk 11 above)
Comment (BhagyashriSSavant) [reply Bhagyashri/first_assignment/css/style.css]: noted

File: Bhagyashri/Correction/css/style.css (hunk 7 above)
Comment (BhagyashriSSavant) [reply Bhagyashri/Correction/css/style.css]: Have given
 min-height:70px;
	 width:60%;

if write width:100%
then image is not fixed at mentioned position.

**Pull request closed without merging**